app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,20 @@
 import * as R from "ramda";
 
+const convertStringToCamelCase = string =>
+  string
+    .toLowerCase()
+    .split(" ")
+    .map((word, index) =>
+      index === 0
+        ? word
+        : word[0].toUpperCase() + word.substring(1, word.length)
+    )
+    .join("");
```

Comment:
So what if I gave `Ux-Designer` as the string? My expectation is that it will return `uxDesigner`. If this method cannot match my expectation, then refactor accordingly to make it match.

Reply:
I believe this can be solved by allowing only letters and numbers to be present in the string, using the `replace` method. So, modifying the `convertStringToCamelCase` like the following should solve this:

```
const convertStringToCamelCase = string =>
  string
    .toLowerCase()
    .replace(/[^a-z0-9]/g, " ")
    .split(" ")
    .map((word, index) =>
      index === 0
        ? word
        : word[0].toUpperCase() + word.substring(1, word.length)
    )
    .join("");
```

This would replace all the non-letters and non-numbers of the string with a space.